modules/readme/Makefile

```diff
@@ -8,27 +8,47 @@ export README_TEMPLATE_REPO_REMOTE ?= $(shell [ -d .git ] && git remote get-url
 # Parse https://github.com/...
 ifneq (,$(findstring https://github.com/,$(README_TEMPLATE_REPO_REMOTE)))
 URL_NO_PROTOCOL := $(subst https://github.com/,,$(README_TEMPLATE_REPO_REMOTE))
-README_TEMPLATE_REPO_ORG ?= $(firstword $(subst /, ,$(URL_NO_PROTOCOL)))
+export README_TEMPLATE_REPO_ORG ?= $(firstword $(subst /, ,$(URL_NO_PROTOCOL)))
 endif
 
 # Parse git@github.com:...
 ifneq (,$(findstring git@github.com:,$(README_TEMPLATE_REPO_REMOTE)))
 URL_NO_GIT := $(subst git@github.com:,,$(README_TEMPLATE_REPO_REMOTE))
-README_TEMPLATE_REPO_ORG ?= $(firstword $(subst /, ,$(URL_NO_GIT)))
+export README_TEMPLATE_REPO_ORG ?= $(firstword $(subst /, ,$(URL_NO_GIT)))
 endif
 
 export README_TEMPLATE_REPO ?= .github
-export README_TEMPLATE_REPO_REF ?= blob/main
+export README_TEMPLATE_REPO_REF ?= main
 export README_TEMPLATE_REPO_PATH ?= README.md.gotmpl
-export README_TEMPLATE_REPO_URL ?= https://github.com/$(README_TEMPLATE_REPO_ORG)/$(README_TEMPLATE_REPO)/$(README_TEMPLATE_REPO_REF)/$(README_TEMPLATE_REPO_PATH)?raw=true
+export README_TEMPLATE_REPO_URL ?= https://raw.githubusercontent.com/$${README_GITHUB_ORG}/$(README_TEMPLATE_REPO)/$(README_TEMPLATE_REPO_REF)/$(README_TEMPLATE_REPO_PATH)
 export README_TEMPLATE_FILE ?= $(BUILD_HARNESS_PATH)/templates/README.md.gotmpl
-export README_TEMPLATE_YAML := $(if $(findstring http,$(README_YAML)),$(README_YAML),$(BUILD_HARNESS_PATH)/templates/$(README_YAML))
-export README_INCLUDES ?= $(file://$(shell pwd)/?type=text/plain)
```

Comment:
@Nuru this is needed by the `README.md.gotmpl` which uses it as the base for including other files (e.g. from `docs/`)

Reply:
With this syntax, `README_INCLUDES` is always empty. Probably meant for it to be automatically including `$PWD` but since it never worked, I would rather just remove it from the template.

- added in #100